src/AvaloniaShellApp/MorganStanley.ComposeUI.AvaloniaShellApp.sln

```diff
@@ -0,0 +1,75 @@
+﻿# Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License"). You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0. See the NOTICE file distributed with this work for additional information regarding copyright ownership. Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
```

Comment:
Is this solution different from the one in `src`?

Reply:
No, it is the basically same. I'll remove it.

Reply:
but let me do it in the next Jira.